Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,122 @@
+- I explored ahead of my team(s) and helped build a technical direction to achieve their mission. I did some exploration solo and some exploration with the team. We experimented with different approaches. We discovered risks and accelerators. We updated the plan when we uncovered new risks/accelerators.
+- I helped a team reduce complexity and risk through sound architectural thinking resulting in better outcomes.
+- I developed a plan to evolve the architecture of a particularly difficult/risky/ambigious part of the core components of Octopus iterating in the open with other people using techniques like RFCs and presentations/breakouts at R&D Weekly and RADAR sessions.
+- I worked with teams to manage multiple conflicting priorities, navigated difficult tradeoffs, and helped sequence work resulting in the best collective outcome.
 - I collaborated with another principal engineer to ensure that a feature I was helping a team with would support a feature I knew another team was planning.
+- I developed a section of the Octopus target architecture, which required a thorough understanding of the problem space, proposing potential solutions, and offering a strong and informed opinion on the best solution.
+- I contributed to building blocks or core technologies used by a variety of teams.
 
 </details>
 
 ## 🌱 Teaching and Mentoring
 
 - I am an expert mentor and teacher.
+- I am curious, listen, and encourage healthy conflict.
+- I allow myself to be persuaded by great ideas.
 - I create clarity by communicating complex things in simple and accessible ways.
 - I understand my broadest impact comes from empowering those around me.
 - I am an expert at designing and reviewing software architecture and code.
-- I help teams and individuals understand our technical strategy, and the things we are doing to progress it.
+- I help teams and individuals understand our technical strategy, how it connects to our product strategy, and the things we are doing to progress it.
 - I regularly provide training and support for new methods, tools, and patterns.
-- I collaborate with managers to identify and find ways to fill any gaps in technical skills required in my teams.
-- I am curious, listen, and encourage debate.
-- I allow myself to be persuaded by great ideas.
+- I actively share knowledge relevant to multiple teams.
+- I collaborate with managers and lead software engineers to identify and find ways to fill any gaps in technical skills required in my teams.
 
 <details>
 <summary>Examples</summary>
 
 - I was consistently in demand for design and code review.
-- I am actively mentoring multiple Octopus engineers.
+- I became aware that I was a bottleneck and worked proactively with managers to uplift and empower other people to make the team more effective.
+- I am actively mentoring multiple Octopus engineers, including a Lead Software Engineer aspiring to become a Principal Software Engineer.
```

Comment:
My experience in heading toward L5 was that there's not that much space in the org for new PSEs. Is the intent of explicitly mentioning an L4 with L5 aspirations intended to be representative of the level of mentoring we should be capable of? Or is it more literal and direct?

My concern here is that if we expect every L5 to be mentoring and stoking the ambitions of (at least) one L4, but we don't have any "space" to actually be promoting any of them into the role, it could be a recipe for sadness and frustration.